packages/documentation/src/content/docs/concepts/accounting/accounts-and-transfers.md

```diff
@@ -12,8 +12,8 @@ A liquidity account may only hold a positive balance. Rafiki enforces that its t
 
 There is one liquidity account for each of the following resource:
 
-- Asset
-- Peer
+- [Asset](/reference/glossary#asset)
```

Comment:
might be good to have this in a separate PR so the writers can also take a glance at it

Reply:
Hi @melissahenderson @tselit , would you mind having a look at the updates I made to accounts and transfers?
Let me know if it is ok reviewing from the link below, or whether you want me to create a separate PR?
https://github.com/interledger/rafiki/pull/2776/files#r1686775804

Thank you!

Reply:
Preview of the actual changes:
https://deploy-preview-2776--brilliant-pasca-3e80ec.netlify.app/concepts/accounting/accounts-and-transfers/